taxonomy-editor-frontend/src/pages/search/index.jsx

```diff
@@ -50,18 +52,18 @@ const SearchNode = () => {
                       }}
                     fullWidth
                     onKeyDown={(e) => {
-                      if (e.key === "Enter" && !isDisabled) {
-                        navigate(`/results?query=${query}`)
+                      if (e.key === "Enter" && query.trim().length !== 0) {
+                        navigate(`/search?query=${query}`)
                       }
                     }}
                     onChange = {event => {
-                        if (event.target.value === "") setIsDisabled(true)
-                        else setIsDisabled(false)
-                        setQuery(event.target.value)
+                        setQuery(event.target.value.trim())
                     }}
                     value={query} />
             </Grid>
         </Container>
+        {queryString !== null && <SearchResults query={queryString}/>}
```

Comment:
what about receiving the data, instead of receiving a queryString

Reply:
Ah, since we're using a useFetch hook, it might be difficult to send the data. Conditionally calling useFetch isn't allowed in React I think.

Reply:
ok, let's keep it like this for now